### PR TITLE
Syncing gitbook information with README

### PR DESCRIPTION
**Pull request by har00ga**
Related to: #4283

---

### Description

Updated to match https://prysmaticlabs.gitbook.io/prysm/

Please give the final README a thorough review. A number of sections have been modified, it's a good chance to check for [in]consistencies. Thanks :slightly_smiling_face:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,194 +33,182 @@ Participation is now open to the public for our Ethereum 2.0 phase 0 testnet rel
...
-1. Open up two terminal windows. In the first, issue the command:
+Open up two terminal windows. In the first, issue the command:
 
-```
+```text
 bazel run //beacon-chain -- \
 --no-genesis-delay \
 --bootstrap-node= \
---deposit-contract $(curl https://prylabs.net/contract) \
+--deposit-contract 0xD775140349E6A5D12524C6ccc3d6A1d4519D4029 \
```
Comment (rauljordan) [inline README.md]: can we keep the curl stuff?

**Comment by rauljordan** [comment]
hi @har00ga do you have some time to wrap up the feedback soon? Thanks so much

File: README.md (hunk 1 above)
Comment (har00ga) [reply README.md]: Reverted :)

**Comment by har00ga** [comment]
Sorry again about the delay, holidays were a mess! Let me know if there's anything else to revise

**Pull request merged**